app/v4/controllers/RetrieveCalculationController.scala

```diff
@@ -63,6 +65,14 @@ class RetrieveCalculationController @Inject() (val authService: EnrolmentsAuthSe
         RequestHandler
           .withValidator(validator)
           .withService(service.retrieveCalculation)
+          .withAuditing(AuditHandler(
+            auditService,
+            auditType = "RetrieveATaxCalculation",
+            transactionName = "retrieve-a-tax-calculation",
+            apiVersion = Version.from(request, orElse = Version4),
```

Comment:
The version will always be in the request. The `orElse` suggests that it might not be (which will puzzle future readers) and in the case where we route different versioned requests to shared controllers this could silently audit the wrong version.
```suggestion
            apiVersion = Version(request),
```

Likewise for v5.